src/utils.jl

```diff
@@ -498,6 +498,14 @@ function make_fillrange_from_ribbon(kw::KW)
     kw[:fillrange] = make_fillrange_side(y, rib1), make_fillrange_side(y, rib2)
 end
 
+#turn tuple of fillranges to one path
```

Comment:
If this is only meant to be used by the plotlyjs backend I'd think it makes sense to keep it there.